laravel/tests/Feature/AuthenticationTest.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace Tests\Feature;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Foundation\Testing\WithoutMiddleware;
+use App\Models\User;
+
+use Tests\TestCase;
+
+class AuthenticationTest extends TestCase
+{
+    use RefreshDatabase;
+
+    protected $user = [
+        'name' => 'tester',
+        'email' => 'test@example.com',
+        'password' => "password",
+    ];
+
+    /**
+     * 登録テスト
+     */
+    public function testRegister():void
+    {
+        $this->post('/register', [
+            'name' => $this->user['name'],
+            'email' => $this->user['email'],
+            'password' => $this->user['password'],
+            'password_confirmation' => $this->user['password'],
+        ]);
+        $response = $this->get('/');
+        $response
+            ->assertStatus(200);
+    }
```

Comment:
データが登録されたあと、指定のURIにリダイレクトされているかの確認であれば、以下のようにすると良いかなと思いました 👍 

そして、`$response = $this->get('get')`の部分は`トップページへアクセスできる`といった内容のテストとして切り出すと尚良いかと思いました 👍 

```suggestion
    /**
     * アクセス可能テスト
     */
    public function testAccess(): void
    {
        $response = $this->get('/');
        $response
            ->assertStatus(200);
    }

    /**
     * 登録テスト
     */
    public function testRegister(): void
    {
        $response = $this->post('/register', [
            'name' => $this->user['name'],
            'email' => $this->user['email'],
            'password' => $this->user['password'],
            'password_confirmation' => $this->user['password'],
        ]);

        // リダイレクト（302の検証）されているかの確認
        // リダイレクト先（route('home')でログイン後の遷移先URIが取得できる）の確認
        $response->assertStatus(Response::HTTP_FOUND)
            ->assertRedirect(route('home'));
    }
```